src/routes/admin/group.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/prefer-default-export */
+import express, { Response, Request } from 'express';
+import httpCodes from 'http-codes';
+import transactionWrapper from '../transactionWrapper';
+import { handleError } from '../../lib/apiErrorHandler';
+import { groupsByRegion } from '../../services/groups';
+
+const namespace = 'ADMIN:GROUPS';
+const logContext = { namespace };
+
+/**
+   *
+   * @param {Request} req - request
+   * @param {Response} res - response
+   */
+export async function getGroupsByRegion(req: Request, res: Response) {
```

Comment:
Groups can span regions, will that cause any issues?

Reply:
It think it won't (for this application, anyways)